addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/CsrfCountermeasuresScanRuleUnitTest.java

```diff
@@ -36,31 +36,40 @@
 import org.mockito.Mockito;
 import org.parosproxy.paros.core.scanner.Alert;
 import org.parosproxy.paros.core.scanner.Plugin.AlertThreshold;
+import org.parosproxy.paros.model.Model;
+import org.parosproxy.paros.model.OptionsParam;
 import org.parosproxy.paros.network.HttpHeader;
 import org.parosproxy.paros.network.HttpMessage;
 import org.parosproxy.paros.network.HttpRequestHeader;
 import org.parosproxy.paros.network.HttpResponseHeader;
 import org.zaproxy.addon.commonlib.CommonAlertTag;
+import org.zaproxy.zap.extension.anticsrf.AntiCsrfParam;
 import org.zaproxy.zap.extension.anticsrf.ExtensionAntiCSRF;
 import org.zaproxy.zap.utils.ZapXmlConfiguration;
 
 class CsrfCountermeasuresScanRuleUnitTest extends PassiveScannerTest<CsrfCountermeasuresScanRule> {
 
     private ExtensionAntiCSRF extensionAntiCSRFMock;
     private List<String> antiCsrfTokenNames;
+    private AntiCsrfParam antiCsrfParam;
     private HttpMessage msg;
 
     @BeforeEach
     void before() throws URIException {
         antiCsrfTokenNames = new ArrayList<>();
         antiCsrfTokenNames.add("token");
         antiCsrfTokenNames.add("csrfToken");
+        antiCsrfTokenNames.add("csrf-token");
 
         extensionAntiCSRFMock = mock(ExtensionAntiCSRF.class);
         Mockito.lenient()
                 .when(extensionAntiCSRFMock.getAntiCsrfTokenNames())
                 .thenReturn(antiCsrfTokenNames);
-
+        OptionsParam options = Model.getSingleton().getOptionsParam();
+        options.load(new ZapXmlConfiguration());
+        antiCsrfParam = new AntiCsrfParam();
+        options.addParamSet(antiCsrfParam);
+        antiCsrfParam.setPartialMatchingEnabled(false);
```

Comment:
Based on `InfoSessionIdUrlScanRuleUnitTest`, it causes some console/logging messiness but we accepted that before 🤷‍♂️